web_src/css/repo/release-tag.css

```diff
@@ -132,3 +132,8 @@
 .repository.new.release .field .attachment_edit {
   width: 450px !important;
 }
+
+.repository.new.release .markup {
+  min-height: 240px;
+  overflow: visible;
```

Comment:
Why the need to override overflow? Is it coming from Fomantic?

Reply:
Removed.